utils/contributors.js

```diff
@@ -33,6 +33,32 @@ const CONTRIBUTERS = [
         `,
         github: "https://github.com/ankur12-1610",
     },
+    {
+        handle_name: 'Sravya32',
+        name: "Sravya Damarla",
+        image: "/images/Sravya.jpg",
```

Comment:
- Since you deleted the previous `jpg` image. You have to add your changes and not delete the existing sample images or data .
- You have attached an empty file as image. I would suggest you to either remove the image path in that case or provide a proper image file.